bridge/test/utils.js

```diff
@@ -63,6 +65,29 @@ function checkRcpt(tx) {
     checkGas(tx.receipt.gasUsed);
   }
 
+// @param tx is an object returned from the smartcontract tx {
+//  tx: @string,
+//  receipt: {
+//    gasUsed: @number
+//  }
+// }
+// @return BN the gas used multiplied by the gas price
+const getGasUsed = async (tx) => {
+  const gasUsed = new BN(tx.receipt.gasUsed);
+
+  // Obtain gasPrice from the transaction
+  const txWithPrice = await web3.eth.getTransaction(tx.tx);
+  const gasPrice = new BN(txWithPrice.gasPrice);
+
+  return gasUsed.mul(gasPrice);
+}
+
+// @param address string to check the amount of ether
+// @return BN the balance of ether
+const getBalance = async (address) => {
```

Comment:
maybe rename to `getEtherBalanceForAddress`? I think `getBalance` might sound pretty generic

Reply:
I renamed as `getEtherBalance` what do you think?

Reply:
awesome!